### PR TITLE
End of  mobile version

### PR DESCRIPTION
**Pull request by hopeforkoami**
I added the work and the about me sections on my portfolio project based on HTML and CSS

### CODE REVIEW

**Review by ElsonOtake — COMMENTED** [review]
Hi @hopeforkoami, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

Invalid code review request

## Invalid Code Review Request

You have submitted a project, that the principal changes were in the default (main) branch. All changes should be made on the feature branch and not the default branch, this way, CRs can use Github features(inline comments) for proper review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏 🙏 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by hopeforkoami** [comment]
> All changes should be made on the feature branch and not the default branch

@ElsonOtake , thank you . please would you mind guiding me on how to do it on the right way?

**Comment by hopeforkoami** [comment]
> Hi @hopeforkoami,
> 
> While you made a great effort in this project, unfortunately, I cannot proceed to review your code.
> 
> Invalid code review request
> ## Invalid Code Review Request
> 
> You have submitted a project, that the principal changes were in the default (main) branch. All changes should be made on the feature branch and not the default branch, this way, CRs can use Github features(inline comments) for proper review.
> 
> Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏 🙏
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _Invalid Code Review Request **does not count** into the code reviews limit._

please , the changes have been made in the [end_of__mobile_version](https://github.com/hopeforkoami/portofolio_site/tree/end_of__mobile_version)

File: index.html [hunk 1]
```diff
@@ -52,15 +48,186 @@ <h2 class="main_section_h2">I'm a software developer</h2>
...
+              <li class="work_language_item">Ruby on rail</li>
+              <li class="work_language_item">css</li>
+              <li class="work_language_item">JavaScript</li>
+              <li class="work_language_item">html</li>
+            </ul>
+            <button class="work_more_detail_button"> See Project</button>
+          </div>
+        </div>
+        <div class="work_element">
+          <div class="work_image">
```
Comment (MahmoudBakr23) [inline index.html:99]: Required

- Great work with the rest of the design it looks neat. However, please replace this dev with an actual image to follow the semantics and the design

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hello team 😃, 

Great work and effort you put into this project so far.

## Highlights 🚀 

There are some excellent points that you have made perfectly. Keep up! 💪
- Great work with the rest of the design 
- Descriptive pr summary and headline 
- Descriptive readme file
- Correct Linters 

However, there is always room for improvement.

> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. 😄

 ## Required Changes ♻️
_Please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

n/a

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (hopeforkoami) [reply index.html:99]: @MahmoudBakr23 , thank you for your remarks, please there is no image there there is no image in the template . I'm using the template of the figma design. please can I use the images of template 1?

**Review by MaganaAsiati — APPROVED** [review]
# Status: Approved [✔️](https://emojipedia.org/check-mark/)
## Hi @hopeforkoami , I'm @MaganaAsiati , your reviewer for this PR  :wave: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


clap

## Highlights
- descriptive README file  :heavy_check_mark:👨🏻‍💻
- Your commits should start with a capital letter :heavy_check_mark:👨🏻‍💻


## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ ] [OPTIONAL] Please make sure to make your commit messages more descriptive in the future.
- [ ] kindly  Add a descriptive description in your pull request in future which give more information about what you implement in this milestone :heavy_check_mark:👨🏻‍💻
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**